grant-project/zcash-grant-system
# Homepage redesign

Closes #96.

### What This Does

Redesigns the homepage, with some dynamic content showing open RFPs.

### Steps to Test

Look at it, try a bunch of different browsers & sizes to make sure it looks good everywhere. Not pictured in screenshot below, but look at it auth'd and not and make sure it shows a relevant action in the intro.

## Screenshots

![home-desktop](https://user-images.githubusercontent.com/649992/53450046-505dd280-39e9-11e9-99c0-da20c34aa72d.png)
